### ISSUE TITLE
SimplerEnv results of EO-1

### ISSUE THREAD

**Issue by ustcwhy** [SHAILAB-IPEC/EO1#9]
### System Info

Thanks for your wonderful work!

I would like to reproduce the fine-tuning results of EO-1 on SimplerEnv dataset. I adopt the https://huggingface.co/IPEC-COMMUNITY/EO-1-3B as the base model, then follow the provided scripts to conduct fine-tuning on https://huggingface.co/datasets/IPEC-COMMUNITY/bridge_orig_lerobot.

But I observed that the results are quite lower than reported in the paper.
|                                                    | 0                   | 1               | 2         | 3      | 4      | 5         | 6          | 7           | 8       | 9       |
|:---------------------------------------------------|:--------------------|:----------------|:----------|:-------|:-------|:----------|:-----------|:------------|:--------|:--------|
| put_spoon_on_tablecloth/matching_partial           | 0.5833333333333334  | nan             | nan       | 0.167  | nan    | 0.347     | 0.778      | nan         | 0.041   | 0.375   |
| put_spoon_on_tablecloth/matching_entire            | 0.4166666666666667  | nan             | nan       | 0.0    | nan    | 0.125     | 0.472      | nan         | 0.0     | 0.208   |
| put_carrot_on_plate/matching_partial               | 0.625               | nan             | nan       | 0.208  | nan    | 0.528     | 0.278      | nan         | 0.333   | 0.333   |
| put_carrot_on_plate/matching_entire                | 0.5416666666666666  | nan             | nan       | 0.042  | nan    | 0.083     | 0.097      | nan         | 0.0     | 0.25    |
| stack_green_block_on_yellow_block/matching_partial | 0.8333333333333334  | nan             | nan       | 0.083  | nan    | 0.319     | 0.403      | nan         | 0.125   | 0.083   |
| stack_green_block_on_yellow_block/matching_entire  | 0.5416666666666666  | nan             | nan       | 0.0    | nan    | 0.0       | 0.042      | nan         | 0.0     | 0.083   |
| put_eggplant_in_basket/matching_partial            | 0.08333333333333333 | nan             | nan       | 0.0    | nan    | 0.667     | 0.875      | nan         | 0.083   | 0.0     |
| put_eggplant_in_basket/matching_entire             | 0.0                 | nan             | nan       | 0.0    | nan    | 0.431     | 0.569      | nan         | 0.041   | 0.0     |
| ckpt_name                                          | ours                | RT-1(Converged) | RT-1(15%) | RT-1-X | RT-2-X | Octo-Base | Octo-Small | RT-1(begin) | OpenVLA | RoboVLM |

I tried to evaluate the provided checkpoint fine-tuned on SimplerEnv, https://huggingface.co/IPEC-COMMUNITY/eo1-qwen25_vl-bridge
It seems that the results are much better, however, still lower than reported on specific tasks.

|                                                    | 0                   | 1               | 2         | 3      | 4      | 5         | 6          | 7           | 8       | 9       |
|:---------------------------------------------------|:--------------------|:----------------|:----------|:-------|:-------|:----------|:-----------|:------------|:--------|:--------|
| put_spoon_on_tablecloth/matching_partial           | 0.625              | nan             | nan       | 0.167  | nan    | 0.347     | 0.778      | nan         | 0.041   | 0.375   |
| put_spoon_on_tablecloth/matching_entire            | 0.5833333333333334 | nan             | nan       | 0.0    | nan    | 0.125     | 0.472      | nan         | 0.0     | 0.208   |
| put_carrot_on_plate/matching_partial               | 0.75               | nan             | nan       | 0.208  | nan    | 0.528     | 0.278      | nan         | 0.333   | 0.333   |
| put_carrot_on_plate/matching_entire                | 0.75               | nan             | nan       | 0.042  | nan    | 0.083     | 0.097      | nan         | 0.0     | 0.25    |
| stack_green_block_on_yellow_block/matching_partial | 0.875              | nan             | nan       | 0.083  | nan    | 0.319     | 0.403      | nan         | 0.125   | 0.083   |
| stack_green_block_on_yellow_block/matching_entire  | 0.7083333333333334 | nan             | nan       | 0.0    | nan    | 0.0       | 0.042      | nan         | 0.0     | 0.083   |
| put_eggplant_in_basket/matching_partial            | 0.9583333333333334 | nan             | nan       | 0.0    | nan    | 0.667     | 0.875      | nan         | 0.083   | 0.0     |
| put_eggplant_in_basket/matching_entire             | 0.7916666666666666 | nan             | nan       | 0.0    | nan    | 0.431     | 0.569      | nan         | 0.041   | 0.0     |
| ckpt_name                                          | ours                | RT-1(Converged) | RT-1(15%) | RT-1-X | RT-2-X | Octo-Base | Octo-Small | RT-1(begin) | OpenVLA | RoboVLM |

My fine-tuning scripts is 

```
GPUS=8
ACCELERATE_ARGS="--main_process_ip=$MASTER_ADDR --main_process_port=$MASTER_PORT \
  --num_machines 1 --machine_rank 0 --num_processes=${GPUS} --multi_gpu"

dataset=/home/hyw/eo1/experiments/3_simpler/data-bridge.yaml
accelerate launch $ACCELERATE_ARGS /home/hyw//eo1/scripts/train.py \
    --output_dir $OUTPUT_DIR \
    --pack_dataset False \
    --model-name-or-path IPEC-COMMUNITY/EO-1-3B \
    --chat_template /home/hyw/eo1/scripts/chat_template.json \
    --data-path ${dataset} \
    --chunk-size 4 \
    --dataloader-num-workers 8 \
    --freeze-vision-tower False \
    --freeze-llm False \
    --freeze-merger False \
    --bf16 True \
    --tf32 True \
    --fp16 False \
    --num-train-epochs 20 \
    --per-device-train-batch-size 128 \
    --learning-rate 1e-4 \
    --merger-lr 1e-4 \
    --vision-lr 2e-5 \
    --weight-decay 0.1 \
    --warmup-ratio 0.03 \
    --lr-scheduler-type cosine \
    --gradient-checkpointing True \
    --save-strategy steps \
    --logging-steps 1 \
    --save-steps 5000 \
    --save-total-limit 3 \
    --report-to none \
    --run-name ${run_name} \
    --attn-implementation flash_attention_2 | tee train.log
```

Could you provide more details with regard to the fine-tuning on SimplerEnv? For example, the base model, learning rate, training epochs, etc.

### Reproduction

See above.

### Expected behavior

See above.
